lib/amazonka/src/Amazonka/Send.hs

```diff
@@ -0,0 +1,156 @@
+module Amazonka.Send
+  ( send,
+    sendEither,
+    paginate,
+    paginateEither,
+    await,
+    awaitEither,
+    sendUnsigned,
+    sendUnsignedEither,
+  )
+where
+
+import Amazonka.Env (Env, Env')
+import Amazonka.Core (AWSPager, AWSRequest, AWSResponse, Error)
+import qualified Amazonka.HTTP as HTTP
+import qualified Amazonka.Pager as Pager
+import Amazonka.Prelude
+import qualified Amazonka.Waiter as Waiter
+import qualified Control.Exception as Exception
+import Data.Conduit (ConduitM)
+import qualified Data.Conduit as Conduit
+import qualified Network.HTTP.Client as Client
+
+-- | Send a request, returning the associated response if successful.
+--
+-- See 'send'.
+sendEither ::
+  ( MonadResource m,
+    AWSRequest a
+  ) =>
+  Env ->
+  a ->
+  m (Either Error (AWSResponse a))
+sendEither env =
+  fmap (second Client.responseBody) . HTTP.retryRequest env
+
+-- | Send a request, returning the associated response if successful.
+--
+-- Errors are thrown in 'IO'.
+--
+-- See 'sendEither'.
+send ::
+  ( MonadResource m,
+    AWSRequest a
+  ) =>
+  Env ->
+  a ->
+  m (AWSResponse a)
+send env =
+  sendEither env >=> hoistEither
+
+-- | Make a request without signing it. You will almost never need to
+-- do this, but some authentication methods
+-- (e.g. @sts:AssumeRoleWithWebIentity@ and @sso:GetRoleCredentials@)
```

Comment:
Normally this wouldn't matter but it might affect searchability

```suggestion
-- (e.g. @sts:AssumeRoleWithWebIdentity@ and @sso:GetRoleCredentials@)
```

Reply:
Good catch.